src/read.c

```diff
@@ -3541,6 +3541,10 @@ static avifResult avifParseMovieBox(avifDecoderData * data,
 
         AVIF_CHECKERR(avifROStreamSkip(&s, header.size), AVIF_RESULT_BMFF_PARSE_FAILED);
     }
+    if (data->tracks.count == 0) {
```

Comment:
It would be good to add a comment to indicate that this simple check depends on the fact that there is at most one "moov" box:

> Box Type: `'moov'`
> Container: File
> Mandatory: Yes
> Quantity: Exactly one

The check I suggested does not depend on this requirement.

Reply:
I have updated the code to use a local boolean like you suggested earlier.